### ISSUE TITLE
Radio/checkbox target size when no label

### ISSUE THREAD

**Issue by oddvernes** [equinor/design-system#1631]
**Describe the bug**

Target size is smaller than what is required under WCAG

**Steps to reproduce the bug**

1. Add radio or checkbox with no label prop and observe size of clickable area

**Expected behavior**

the clickable area should be the same size as when label is present, 48x48 normal and 32x32 compact
